test/build.bats

```diff
@@ -137,6 +137,31 @@ load common
     rm -Rf --one-file-system "$tar" "$img"
 }
 
+@test 'ch-pull2tar' {
+    scope standard
+    need_docker
+    tag='alpine:latest'
+    tar="${ch_tardir}/${tag}.tar.gz"
+    ch-pull2tar "$tag" "$ch_tardir"
+    [[ $status -eq 0 ]]
+    [[ -e $tar ]] 
+    rm "${ch_tardir}/${tag}.tar.gz"
```

Comment:
Explain in comment why we're deleting the image instead of keeping it around to test.

Or maybe we should keep these around to test? They could be renamed to avoid clashing with the other tarball from the same source. Does the existing framework pick up tarballs added to `$ch_tardir` like this?

Reply:
Commit 4f3b126 adds a comment similar to that of `ch-build2dir`. I am uncertain that it is the correct approach.

I am unsure if the image directories from `ch-pull2dir` and `ch-build2dir` should persist after these specific tests. I think they should be tested with `ch-run` at the very least before being removed. What do you think?

Yes, the existing framework does pick up tarballs with `tarball=$ch_tardir/%(tag)s.tar.gz`.